packages/dynamoose/lib/Table/index.ts

```diff
@@ -257,6 +257,8 @@ export class Table extends InternalPropertiesClass<TableInternalProperties> {
 
 					this.getInternalProperties(internalProperties).pendingTasks.forEach((task) => task.reject(error));
 					this.getInternalProperties(internalProperties).pendingTasks = [];
+
+					throw error;
```

Comment:
what if I added one more line of code and it looked like this?

Reply:
Not sure what you mean here.

Reply:
If you look at the existing unit tests, there are already test cases for errors in that function. Here, if I prevent the error from occurring with a catch statement, I have to rewrite the related test cases. However, this may cause problems for other projects that already import that library. But since all I want is to raise a reject on the error so that the pendingTasks don't get stuck forever, I've made sure to raise the error again at the end of that catch statement. @fishcharlie